ingest/checkpoint_change_reader_test.go

```
@@ -757,7 +757,7 @@ func entryAccount(t xdr.BucketEntryType, id string, balance uint32) xdr.BucketEn
 			Type: xdr.BucketEntryTypeDeadentry,
 			DeadEntry: &xdr.LedgerKey{
 				Type:    xdr.LedgerEntryTypeAccount,
-				Account: &xdr.LedgerKeyAccount{xdr.MustAddress(id)},
+				Account: &xdr.LedgerKeyAccount{AccountId: xdr.MustAddress(id)},
```

I think this is good, but one risk is that, if you do:
`MyStruct{foo}` in go, and MyStruct has 2 fields, the code will fail to compile, while if you do named field, it will silently ignore the other fields. This introduces risk if refactoring later and adding fields to the struct. Overall, probably nicer, though.

---

It's not really for looking nicer - it's so that we can ensure that the value get assigned to the correct variable in case we change the variables order in the struct definition. As a sidenote, it also eliminates a linter error..